### ISSUE TITLE
Change the objectives

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/js-objects-this#28]
@gaand says: Access to reference? 

I'm not sure what he means, but hopefully he can clarify.

**Comment by gaand** [comment]
```diff
 ## Objectives

 -   Contrast the defintions of "property", "attribute", and "method"
--   From within a method, reference properties of the same object using `this`.
+-   From within a method, access properties of the same object using `this`.
 
 ## Preparation
```

**Comment by BenGitsCode** [comment]
I believe this has been addressed in aee4fdc20a34ec4754f71b10a61ba094a12fd509
